test/core/sup_tree_core/tmpdir_tracker_test.exs

```diff
@@ -0,0 +1,16 @@
+# Copyright(c) 2015-2019 ACCESS CO., LTD. All rights reserved.
+
+defmodule AntikytheraCore.TmpdirTrackerTest do
+  use Croma.TestCase
+  alias Antikythera.Tmpdir
+  alias AntikytheraCore.TmpdirTracker
```

Comment:
This line is no longer needed. (also `ZipTest`)
`Croma.TestCase` automatically alias the module under test.
For more details see [doc](https://hexdocs.pm/croma/Croma.TestCase.html).

Reply:
Applied alias in bf9eb99 .